src/main/scala/cromwell/binding/Scope.scala

```diff
@@ -26,33 +27,50 @@ object Scope {
     def getChildrenList(ast: Ast): Seq[Ast] = Option(ast.getAttribute("body")) collect {
       case list: AstList => list.asScala.toVector
     } getOrElse Seq.empty collect {
-      case ast: Ast => ast
+      case node: Ast
+        if node.getName == AstNodeName.Call ||
+          node.getName == AstNodeName.Scatter =>
+        node
+    }
```

Comment:
@mcovarr No worries, will do a single line. It was an over optimization for future `Scope` types.

@Horneth Yeah, I noticed your `generateTree` was generic. But when merging my code with your prior changes, I didn't like the style(?) of returning `None` to mean "whoops, I saw something I didn't expect". I believe we approached the original problem from different perspectives-- I was differentiating the "levels" of nodes/scopes as much as possible, where you appeared to keep them as generic as possible. Right now, I don't even like how `setChildScopes` is invoked on `Call` nodes, but need to stop iterating on this. :sweat_smile: Yours ultimately is a K.I.S.S. approach, where I ended up fighting with the type system MUCH more than I should have. Still I'll leave it to a third reviewer ( @scottfrazer ?) to decide what direction this commit, and ultimately the internal code should take.